src/WordPress/Blueprints/BlueprintParser.php

```diff
@@ -152,14 +153,15 @@ private function findSubErrorForSpecificAnyOfOption( \Swaggest\JsonSchema\Except
 			}
 		}
 		if ( ! $e->subErrors ) {
-			return;
+			return $e->error;
 		}
 		foreach ( $e->subErrors as $subError ) {
 			$subError = findSubErrorForSpecificAnyOfOption( $subError, $anyOfRef );
 			if ( $subError !== null ) {
 				return $subError;
 			}
 		}
+        return $e;
```

Comment:
Nitpick: This is indented with spaces while the rest of this file is indented with tabs

Reply:
The PHPStorm autolinter seems to be underconfigured. As this is not the only comment regarding linting, there is a choice for you here - option a. we close our eyes, accept tiny linting mishaps, and configure and run a linter through the whole project in a separate PR, b. we do it now, and make this a massive "set all standards at once PR". I can go either way. The choice is yours.

Reply:
> The PHPStorm autolinter seems to be underconfigured.

You could configure it to use tabs instead of spaces, that would do the trick for now. And I agree, let's add linter in a subsequent PR.

Reply:
Sounds like a plan!